lib/screens/displayphotos_screen.dart

```diff
@@ -100,6 +102,55 @@ class _DisplayPhotoScreenState extends State<DisplayPhotoScreen> {
       );
     }
 
+    postData(filepaths, apiUrl) async {
+      try {
+        var request = http.MultipartRequest('POST', Uri.parse(apiUrl));
+
+        //TODO: request fields
+        request.fields['userid'] = '1';
+        request.fields['cooking_date'] = '2022-10-22';
+
+        List<http.MultipartFile> newList = <http.MultipartFile>[];
+
+        for (int i = 0; i < filepaths.length; i++) {
+          File imageFile = File(filepaths[i]);
+          var stream = new http.ByteStream(imageFile.openRead());
+          var length = await imageFile.length();
+          var multipartFile = new http.MultipartFile("pictures", stream, length,
+              filename: path.basename(imageFile.path));
+          newList.add(multipartFile);
+        }
+        request.files.addAll(newList);
+
+        var res = await request.send();
+        print(res.toString());
+
+        res.stream.listen((value) {
+          print('value');
+          print(value);
+        });
+        if (res.statusCode == 200) {
+          print('uploaded');
+        } else {
+          print('failed');
+        }
+      } catch (e) {
+        print(e);
+      }
+    }
+
+    //Uploading function
+    void _uploadImage() async {
+      print('continue');
+
+      final String address = "http://127.0.0.1/api";
```

Comment:
Dont statically code api urls. This should be set from some central location so that we can change it.

Reply:
Done. And I update the HttpService. Thanks for the suggestion!